pkg/agent/openflow/pipeline.go

```diff
@@ -229,9 +228,10 @@ var (
 	// ofPortMarkRange takes the 16th bit of register marksReg to indicate if the ofPort number of an interface
 	// is found or not. Its value is 0x1 if yes.
 	ofPortMarkRange = binding.Range{16, 16}
-	// OfTraceflowMarkRange stores dataplaneTag at range 28-31 in marksReg.
-	OfTraceflowMarkRange = binding.Range{28, 31}
-	// ofPortRegRange takes a 32-bit range of register portCacheReg to cache the ofPort number of the interface.
+	// OfTraceflowMarkRange stores dataplaneTag at range 2-7 in DSCP field of IP header.
+	// IPv4/v6 DSCP (Bits 2-7) Field supports exact match only.
+	OfTraceflowMarkRange = binding.Range{2, 7}
```

Comment:
I find this a bit confusing. The DSCP field is bits 2 through 7 of the IP ToS field. I don't think "range 2-7 in DSCP field" makes sense.

We are only using this constant in the code below: 
```go
packetOutBuilder = packetOutBuilder.AddLoadAction(binding.NxmFieldIPTos, uint64(dataplaneTag), OfTraceflowMarkRange)
```
it may make more sense to define the constant locally where it is used, rather than here in the middle of the "register ranges"

BTW, isn't there an OVS field for DSCP we can use directly? `OXM_OF_IP_DSCP`? @wenyingd

Reply:
Renamed to traceflowMarkToSRange, move to a separate section and updated comments.
I prefer to store all ranges in same place, because it is easy to query.

Yes, OXM_OF_IP_DSCP is what we want, will test with this

Reply:
Tested OXM_OF_IP_DSCP, still requires range{0, 5}. We don't need to move to this.